engfrosh_site/management/templates/manage.html

```diff
@@ -11,17 +11,9 @@
       <a href="discord/magic-link"
          class="bg-main dark:bg-main-night rounded-lg p-2">Magic Links</a>
       <a href="users/add" class="bg-main dark:bg-main-night rounded-lg p-2">Add Users</a>
-      <a href="{% url 'add_users_to_discord_guild' %}"
```

Comment:
Same as urls.py, this doesn't match up with the views you removed and is currently very useful.

Reply:
Just reverted that change, accidentally removed the wrong view